core/src/main/java/com/linecorp/armeria/server/logging/AccessLogWriter.java

```diff
@@ -50,7 +55,15 @@ static AccessLogWriter combined() {
      * Returns disabled access log writer.
      */
     static AccessLogWriter disabled() {
-        return requestLog -> { /* No operation. */ };
+        return DISABLED;
+    }
+
+    /**
+     * Returns whether this writer is the disabled access log writer.
+     *
+     */
+    default boolean isDisabled() {
```

Comment:
How about doing `accessLogWriter != AccessLogWriter.DISABLED` this instead of introducing this new public API?

Reply:
done!